### DISCUSSION TITLE
Allow to filter `+1` comments from issue/pr/discussion subscribtions

### DISCUSSION THREAD

**Discussion by Skgland** [community/community#102146] [Discussions]
### Select Topic Area

Product Feedback

### Body

Being subscribed to many Issues/PRs/Discussions gets annoying due to nonconstructive comments such as

> +1 

which could have been a click on  `👍` / `↑` instead.

I generally also don't want to unsubscribe as I am waiting for resolution/constructive comments.

Someone about to post such a comment could also get a dialog suggesting to add a reaction instead or if they are sure they want to post such an nonconstructive comment.

**Comment by bormm** [comment]
I support this request. How I can push it? Maybe by opening a ticket because I am a paying user.
Its absolutely annoying to get "someone added a comment +1" or "+100" or "+ 99" notifications.

Not only github should stop sending me that as a notification, its also spamming the actual discussion thread.
So maybe we need a comment quality score that is going lower if your comment contains mostly "+1" stuff. If peoples public quality score is is getting lower for each "+1" and higher for using the 👍 / ↑ they will change their behavior automatically.

**Comment by bormm** [comment]
As this is not only about the separate discussions but also about comments/discussions inside issues, I think the label "Issues" need to be added here to have it in multiple categories. Can you add that @Skgland as author of this discussion or can that only be done by github people?

**Reply by Skgland** [reply]
No, it appears I can only change the category not the labels, and I am only able to select one category.

**Reply by Skgland** [reply]
Also there doesn't appear to be an `Issues` Category only `Pull Requests` and `Discussions`

**Reply by Skgland** [reply]
Ah, but there appears to be an `Issues` label belonging to `Projects and Issues`
